tools/src/main/scala/scala/scalanative/build/Build.scala

```diff
@@ -63,22 +64,32 @@ object Build {
       ScalaNative.logLinked(fconfig, linked)
       val optimized = ScalaNative.optimize(fconfig, linked)
 
+      // clean ll files
       IO.getAll(workdir, "glob:**.ll").foreach(Files.delete)
-      ScalaNative.codegen(fconfig, optimized)
-      val generated = IO.getAll(workdir, "glob:**.ll")
+
+      val generated = ScalaNative.codegen(fconfig, optimized)
 
       val nativelibs   = NativeLib.findNativeLibs(fconfig.classPath, workdir)
       val nativelib    = NativeLib.findNativeLib(nativelibs)
       val unpackedLibs = nativelibs.map(LLVM.unpackNativeCode(_))
 
-      val objectFiles = config.logger.time("Compiling to native code") {
+      val msg = s"Compiling to native code (${targetTripleMsg(config)})"
+      val objectPaths = config.logger.time(msg) {
```

Comment:
I don't see why we should display the target triple here, if it directly comes from the config anyway. Especially, why should we display that, but not all the other config values?

IMO this should be reverted:
```suggestion
      val objectPaths = config.logger.time("Compiling to native code") {
```

Reply:
It is not just about config values but showing the ones that really matter. Currently, we show the `GC`, `lto`, optimization level like `debug` and `release-fast` and other `Dumping` messages. Target triple is very important because that is the native code that is being compiled. It also brings awareness to people using Scala Native and most Scala programmers probably know little about the Clang/llvm platform. The target triple will become even more important in the future if we can target smaller platforms or ones without a JVM that require cross compilation.

```
sbt:scala-native> sandbox/run
[info] Linking (1751 ms)
[info] Checking intermediate code (278 ms)
[info] Dumping intermediate code (linked) (740 ms)
[info] Discovered 924 classes and 6981 methods
[info] Optimizing (debug mode) (2677 ms)
[info] Checking intermediate code (92 ms)
[info] Dumping intermediate code (optimized) (528 ms)
[info] Dumping intermediate code (lowered) (943 ms)
[info] Generating intermediate code (2767 ms)
[info] Produced 8 files
[info] Compiling to native code (default target triple) (904 ms)
[info] Linking native code (immix gc, none lto) (184 ms)
[info] Total (10167 ms)
Hello, World!
[success] Total time: 14 s, completed Dec 20, 2020 7:50:43 AM
```

Reply:
@sjrd I just removed this as you recommended. We can always add it if we get requests and I asked around and it seems I was the most interested in this output. So this should be ready to go.